lib/improver/tests/ensemble_calibration/ensemble_calibration/helper_functions.py

```diff
@@ -52,10 +52,10 @@ def set_up_probability_above_threshold_cube(
     values for the coordinate.
     """
     cube_long_name = (
-        "probability_of_{}_above_threshold".format(phenomenon_standard_name))
+        "probability_of_{}".format(phenomenon_standard_name))
     cube = Cube(data, long_name=cube_long_name,
                 units=phenomenon_units)
-    coord_long_name = "{}_threshold".format(phenomenon_standard_name)
+    coord_long_name = "threshold"
```

Comment:
I've kept the changes as they are, although yes, this needs to be revisited when the support for the probability to percentile conversion is implemented.